digital_comms/ccam.py

```diff
@@ -239,7 +365,7 @@ def demand(self):
 
     @property
     def population_density(self):
-        """Calculate population density
+        """obj: The population density in people per TODO
```

Comment:
"""obj: Population density in persons per square kilometer (km^2)"""

Reply:
Done